src/box/lua/info.c

```diff
@@ -731,6 +731,16 @@ lbox_info_synchro(struct lua_State *L)
 	lua_setfield(L, -2, "busy");
 	luaL_pushuint64(L, queue->promote_greatest_term);
 	lua_setfield(L, -2, "term");
+	if (queue->len == 0) {
+		lua_pushnumber(L, 0);
```

Comment:
See my comment below about `confirm_lag`.

> IMHO, 0 means the entry was just committed.

It will never be exactly 0, if there is an entry in the limbo. 

The user can also look at `box.synchro.queue.len`. But I think that if the user sees that `age` is zero, he should simply look at the `confirm_lag`.

Reply:
I agree with @CuriousGeorgiy here. Better show it as 0 than show it from time to time.

Reply:
Ok, let it be 0. I don't mind